tgui/packages/tgui/components/Box.tsx

```diff
@@ -264,15 +263,5 @@ export const Box: SFC<BoxProps> = (props: BoxProps) => {
       : computeBoxClassName(rest);
   const computedProps = computeBoxProps(rest);
   // Render a wrapper element
-  return createVNode(
-    VNodeFlags.HtmlElement,
-    as,
-    computedClassName,
-    children,
-    ChildFlags.UnknownChildren,
-    computedProps,
-    undefined
-  );
+  return createElement(as, computedClassName, children, undefined);
```

Comment:
i'm suspicious of this block in particular

Reply:
so this would instead be `createElement(as, { className: computedClassName }, children)`